pennylane/devices/__init__.py

```diff
@@ -34,6 +34,7 @@
     default_gaussian
     default_mixed
     default_qutrit
+    qutrit_mixed
```

Comment:
This section of the init file is for the devices, not for submodules. [This section](https://github.com/PennyLaneAI/pennylane/blob/cb6f1cdb8cda09eae9113284eb12b9dde18544a2/pennylane/devices/__init__.py#L129) is an example of how submodules should be added for docs. I don't think we need to start advertising qutrit mixed state stuff just yet and can wait until more files are present in the `qutrit_mixed` submodule.